### PR TITLE
Add xUnit to Time Bar Chart

### PR DESCRIPTION
**Pull request by felixmichel**
Add the option to add an xUnit to a Time Bar. In the config file, one can add an xUnit and then the unit will rendered at the last available bar.
Bildschirmfoto 2021-09-07 um 11 57 49

### CODE REVIEW

File: src/components/Chart/Bars.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ import {
   deduplicate,
   unsafeDatumFn,
   subsup,
-  getTextColor
+  getTextColor,
+  getLastItemFromArray
```
Comment (annatraussnig) [inline src/components/Chart/Bars.js:23]: `getLastItemFromArray`is a bit of a ~mouthful~ readful (plus it's a little misleading, you don't actually get anything back). I'd stick to `last` or `isLast`/ `isLastItem` is `last`is a little too terse for your taste.

File: src/components/Chart/XAxis.js [hunk 2]
```diff
@@ -24,7 +25,9 @@ const XAxis = ({
   x,
   xDomain,
   format,
-  strong
+  strong,
+  xUnit,
+  yScaleChangeDirection
```
Comment (annatraussnig) [inline src/components/Chart/XAxis.js:30]: still the old property name here ;)

**Review by annatraussnig — CHANGES_REQUESTED** [review]

File: src/components/Chart/XAxis.js (hunk 2 above)
Comment (felixmichel) [reply src/components/Chart/XAxis.js]: fixed

**Review by annatraussnig — APPROVED** [review]
Rock'n'roll!

**Pull request merged**